openfe/setup/atom_mapping/lomap_mapper.py

```diff
@@ -14,9 +14,13 @@ class LomapAtomMapper(LigandAtomMapper):
     time: int
     threed: bool
     max3d: float
+    element_change: bool
+    seed: str
```

Comment:
In most cases the likelihood is that the mapper will be used in sucessing on multiple similar molecules. Could it be possible to add an option to populate and cache the seed into the mapper?

Reply:
you could probably make the Mapper slowly learn the seed, but it wouldn't be a good way to do this (the learning rate would depend a lot on the order you used the mapper).  I'm going to say out of scope?

Reply:
Sure (it's why I didn't block over it 🙃 ), that being said it would be a good one to at least document somewhere? Possibly in ExampleNotebooks